src/tx.h

```diff
@@ -179,7 +180,7 @@ class CBaseTx {
     virtual uint256 GetHash() const = 0;
     virtual uint64_t GetFee() const = 0;
     virtual double GetPriority() const = 0;
-    virtual uint256 SignatureHash() const = 0;
+    virtual uint256 SignatureHash(bool reCalculate = false) const = 0;
```

Comment:
"reCalculate" should be a word，write as "recalculate"?

Reply:
done